style.css

```diff
@@ -24,46 +26,179 @@ body {
 .header {
   display: flex;
   height: 50px;
+  padding: 12px;
+  padding-left: 2%;
+  padding-right: 4%;
   width: 100%;
   align-items: center;
+  justify-content: space-between;
 }
 
 .logo {
   text-decoration: none;
   font-weight: 700;
   font-size: 18px;
   color: #6070ff;
-  margin: 3%;
-  margin-top: 2.5%;
-  width: 76%;
-  height: 50px;
   cursor: pointer;
-  float: left;
-  transition: 0.7s;
+  margin-left: 12px;
 }
 
-.logo:hover {
-  font-size: 19px;
-  transform: translate(0, -1px);
+.menu-list {
+  margin-left: -200%;
+  transition: all 0.3s ease-in-out;
+  background-color: #fff;
 }
 
-.nav {
-  flex: 1;
-  text-align: right;
-  margin-right: 12px;
+ul {
+  display: flex;
+  list-style-type: none;
 }
 
-.menu-list {
+nav ul {
   display: none;
 }
 
-.menu {
+.skills a {
+  text-decoration: none;
+}
+
+.skills .extra {
+  text-decoration: none;
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+}
+
+.extra a {
+  background: #e5e5e5;
+  text-decoration: none;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-content: center;
+  gap: 2%;
+  height: 65px;
+  margin-right: 12px;
+}
+
+.menu-list.active {
+  scale: 100%;
```

Comment:
- [ ] Optional: It is highly recommended to implement this. Your design currently overlaps the background, You can add a backdrop so that your background is blurred as per the Figma design.

![2022-06-21 (2)](https://user-images.githubusercontent.com/70269507/174815692-43a68127-2c5f-45e1-b088-9a1bf56a4aa0.png)

Reply:
Thank you @sja-thedude, I have now studied the blur effect in css because of your feedback.
I appreciate